english/work-out/weights/weights.md

```diff
@@ -3,4 +3,6 @@ thinking "I remember how easy this was in high-school...".
 
 After getting under the bar, you quickly realize you are no longer as strong as
 you were in highschool. Struggling to press the bar back up, you concede to gravity
-and sheepishly roll the bar down off yourself.
\ No newline at end of file
+and sheepishly roll the bar down off yourself.
+
+or you go with the [personal trainer](https://github.com/cvillere/create-your-own-adventure/blob/local-user/english/work-out/personal-trainer/personal_trainer.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.